src/results_json.h

```diff
@@ -1,8 +1,9 @@
 #pragma once
 
+#include "hyperloglog.h"
```

Comment:
Same here, I don't think anything is even using definitions from `hyperloglog.h` here